lib/rucio/api/subscription.py

```diff
@@ -35,6 +35,9 @@
 from rucio.core import subscription
 
 
+SubscriptionRuleState = namedtuple('SubscriptionRuleState', ['account', 'name', 'state', 'count'])
```

Comment:
can you point me to it? I had to replace the KeyedTuple, because that is no more

Reply:
can you point me to it? I had to replace the KeyedTuple, because that is no more

Reply:
https://github.com/rucio/rucio/blob/master/lib/rucio/common/constants.py

--> e.g. it has the FTS States there

Reply:
yeah, this is only used here and does not break the API, so that you can still use

```py3
for row in list_subscription_rule_states():
    print(row.account, row.name, row.state, row.count)
```
or similar code. Contrary to `FTS_STATES`, this isn't constant so I would not put it into the constants module